File: php/Human.php
<?php

class Human
{
	private $nama;
	private $nik;
	private $jenis_kelamin;
	
	public function __construct(){
		$this->nama = "-";
		$this->nik = "-";
		$this->jenis_kelamin = '-';
	}
	public function setNama($nama){
		$this->nama = $nama;
	}
	public function getNama(){
		return $this->nama;
	}
	public function setNik($nik){
		$this->nik = $nik;
	}
	public function getNik(){
		return $this->nik;
	}
	public function setJenis_kelamin($jenis_kelamin){
		$this->jenis_kelamin = $jenis_kelamin;
	}
	public function getJenis_kelamin(){
		return $this->jenis_kelamin;
	}
}
?>

File: cpp/Human.cpp
class Human
{
	private:
		string nama;
		string nik;
		char jenis_kelamin;
	public:
		Human(){		// set nilai atribut default
			this->nama = "-";
			this->nik = "-";
			this->jenis_kelamin = '-';
		}
		void setNama(string nama){
			this->nama = nama;
		}
		string getNama(){
			return this->nama;
		}
		void setNik(string nik){
			this->nik = nik;
		}
		string getNik(){
			return this->nik;
		}
		void setJenis_kelamin(char jenis_kelamin){
			this->jenis_kelamin = jenis_kelamin;
		}
		char getJenis_kelamin(){
			return this->jenis_kelamin;
		}
};